iOS-sample-app/Showcases/Screen Views/Modals/ProgrammaticModalPresentingViewController.swift

```diff
@@ -0,0 +1,49 @@
+
+import UIKit
+import ContentSquare
+
+// This class presents a modal programmatically. The modal is implemented in ProgrammaticModalViewController.
+//
+// We need to implement three screen views here:
+// - A screen view for this modal presenter when it appears on screen
+// - A screen view for the modal when it is presented on screen
+// - A screen view for this modal presenter when the modal is dismissed and this presenter is displayed again
```

Comment:
We need to trigger screen views at three different moments :
- when this ViewController appears on screen
- when the modal appears on screen
- when this ViewController re-appears on screen after closing the modal